### PR TITLE
Sane doc reports

### PR DESCRIPTION
**Pull request by agamm**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: #1836

## Description
- Updates the sane-doc-reports docker 
- Adds logo arguments

## Screenshots
`--` 

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [x] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

File: Packs/Base/Scripts/SaneDocReportV2/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+- Updating sane doc reports and adding customer logo.
```
Comment (ronykoz) [inline Packs/Base/Scripts/SaneDocReportV2/CHANGELOG.md:2]: Please use the new release notes format
demisto-sdk update-release-notes

File: Packs/Base/Scripts/SaneDocReportV2/SaneDocReportV2.py [hunk 2]
```diff
@@ -12,16 +12,19 @@
 try:
     sane_json_b64 = demisto.args().get('sane_docx_report_base64', '').encode(
         'utf-8')
-    orientation = demisto.args().get('orientation', 'portrait').encode(
-        'utf-8')
-    paper_size = demisto.args().get('paperSize', 'A4').encode(
-        'utf-8')
+    orientation = demisto.args().get('orientation', 'portrait')
```
Comment (ronykoz) [inline Packs/Base/Scripts/SaneDocReportV2/SaneDocReportV2.py:15]: why did you remove the encoding?

File: Packs/Base/Scripts/SaneDocReportV2/CHANGELOG.md (hunk 1 above)
Comment (agamm) [reply Packs/Base/Scripts/SaneDocReportV2/CHANGELOG.md:2]:
```
Starting to update release notes.
No changes were detected.
```
?

File: Packs/Base/Scripts/SaneDocReportV2/SaneDocReportV2.py (hunk 2 above)
Comment (agamm) [reply Packs/Base/Scripts/SaneDocReportV2/SaneDocReportV2.py:15]: I don't think there was a need, unless you think there are times it will be affected?

File: Packs/Base/Scripts/SaneDocReportV2/CHANGELOG.md (hunk 1 above)
Comment (ronykoz) [reply Packs/Base/Scripts/SaneDocReportV2/CHANGELOG.md:2]: @amshamah419 can you take a look at this?

File: Packs/Base/Scripts/SaneDocReportV2/SaneDocReportV2.py (hunk 2 above)
Comment (ronykoz) [reply Packs/Base/Scripts/SaneDocReportV2/SaneDocReportV2.py:15]: tbh i'm not 100% sure, let's add this to be on the safe side

File: Packs/Base/Scripts/SaneDocReportV2/CHANGELOG.md (hunk 1 above)
Comment (amshamah419) [reply Packs/Base/Scripts/SaneDocReportV2/CHANGELOG.md:2]: @agamm Can you try updating your `demisto-sdk`?

Comment (agamm) [reply Packs/Base/Scripts/SaneDocReportV2/CHANGELOG.md:2]: @ronykoz @amshamah419  I updated before:
image

(Already tried: `pip3 install --upgrade demisto-sdk`) anything I need to change in the code / make changes ?

File: Packs/Base/Scripts/SaneDocReportV2/SaneDocReportV2.py (hunk 2 above)
Comment (agamm) [reply Packs/Base/Scripts/SaneDocReportV2/SaneDocReportV2.py:15]: Is this a convention in content? 
I remember that it works ok in pdf reports:
image
(at least most of them). 
Maybe @glicht do you have an idea why we need to encode utf8?

**Comment by agamm** [comment]
Thanks @amshamah419 !

File: Packs/Base/Scripts/SaneDocReportV2/SaneDocReportV2.py (hunk 2 above)
Comment (ronykoz) [reply Packs/Base/Scripts/SaneDocReportV2/SaneDocReportV2.py:15]: If the docker image is python3 we don't need it
